### ISSUE TITLE
Charging Point Radar Booster Upgrade

### ISSUE THREAD

**Issue by Haravikk** [Malcolm-Q/LC-LateGameUpgrades#179]
When unlocked, the charging point increases the price of the radar booster, but allows players to charge their battery operated items at it without having to return to the ship.

To balance it, the charging point should only be able to charge an item by up to 50% (so an item at 0% would charge to 50%, an item on 50% could go back up to 100% etc.) after which there is a cooldown before another item can be charged. Later upgrades could reduce the cooldown.

Basically the idea is to make the radar booster a more useful item, and allow players to get some charge back without having to go all the way back to the ship, especially on the larger maps. This would be especially useful in darker interiors.

**Comment by WhiteSpike** [comment]
Just to clarify, this would be basically the Coil in the dropship which restores set amount of battery of the item and has a cooldown between each use and this attached to each radar booster?

**Comment by Haravikk** [comment]
> Just to clarify, this would be basically the Coil in the dropship which restores set amount of battery of the item and has a cooldown between each use and this attached to each radar booster?

Yeah that's basically the idea; so you can take it with you into larger facilities and/or on larger maps, so you can recharge items partially without running all the way back to the ship.

Just seems to me like a handy bit of extra utility for the booster that's as useful as you make it; I know they already added a stun feature to it that ups the value quite a bit, but it's still a tricky one to use in practice.

**Comment by Malcolm-Q** [comment]
Moved to discussions https://github.com/Malcolm-Q/LC-LateGameUpgrades/discussions/219
